tasks/queue-workflow/README.md

```diff
@@ -0,0 +1,13 @@
+# @cumulus/queue-workflow
+
+Adds a workflow to a queue
```

Comment:
Nit: Thanks for making the change to add this - I think we need to include a bit more context on the expected input/outputs for integrators, either here, the description strings or referenced to a doc, as we get support questions and/or have gotten feedback that we need more explanation/docs. 

A specific example of what I mean --  "what's a workflowInput", that can be as simple as 'the payload to the workflow to be executed' in the input schema and/or here.

Reply:
Totally fair. Updated the README with all the param details